bug_monitor_add.php

```diff
@@ -85,13 +91,15 @@
 	$g_project_override = $t_bug->project_id;
 }
 
-if( $t_logged_in_user_id == $t_user_id ) {
-	access_ensure_bug_level( config_get( 'monitor_bug_threshold' ), $f_bug_id );
-} else {
-	access_ensure_bug_level( config_get( 'monitor_add_others_bug_threshold' ), $f_bug_id );
-}
+foreach( $t_user_ids as $t_user_id ) {
+	if( $t_logged_in_user_id == $t_user_id ) {
+		access_ensure_bug_level( config_get( 'monitor_bug_threshold' ), $f_bug_id );
+	} else {
+		access_ensure_bug_level( config_get( 'monitor_add_others_bug_threshold' ), $f_bug_id );
+	}
 
-bug_monitor( $f_bug_id, $t_user_id );
+	bug_monitor( $f_bug_id, $t_user_id );
```

Comment:
Maybe have a separate loop to add all monitors.  This way if this loop fails, we end up with no changes, rather than half the changes.